### ISSUE TITLE
Is this the correct repo for mielenosoitukset.fi?

### ISSUE THREAD

**Issue by samimaat** [botsarefuture/mielenosoitukset.fi#1]
Basically the heading's question for the site [mielenosoitukset.fi](https://mielenosoitukset.fi/). Based on the readme this should be it, but the code doesn't look like this is the version that is running at the moment.

It would be useful to add some contact information to the site for improvements! I would also be interested in providing improvements when it comes to the structure of the site and accessibility.

Great work either way!

**Comment by botsarefuture** [comment]
You're right, this isn't the up-to-date version of our code. Frankly I have no idea, what this repo is...

But anyways I can send you a invite to the correct repo, it's private for security reasons.

I'm looking forwards to working with you!

P.S.A. We'd really appreciate your help!
